### PR TITLE
Added rest_jdbc_swarm archetype

### PR DESCRIPTION
**Pull request by heiko-braun**

### CODE REVIEW

**Comment by heiko-braun** [comment]
@cmoulliard Can you take it from here? Maybe we need to disable the unit test because it assumes a postgres database

**Comment by gastaldi** [comment]
You need to add a <module> entry to this quickstart too in archetypes/pom.xml, otherwise it won't be deployed/built

**Comment by gastaldi** [comment]
By entry, I mean a `<module>`

**Comment by heiko-braun** [comment]
Thanks for the hint @gastaldi. I've added it.

File: archetypes/swarm-greenfield-archetype/pom.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <groupId>org.obsidiantoaster.quickstarts</groupId>
+    <artifactId>archetypes</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>swarm-greenfield-archetype</artifactId>
+  <name>Fabric8 :: Archetypes :: swarm-greenfield</name>
+  <packaging>maven-archetype</packaging>
+
+  <description>Creates a new Obsidian :: Quickstart :: Wildfly Swarm Greenfield</description>
```
Comment (gastaldi) [inline archetypes/swarm-greenfield-archetype/pom.xml]: Remove the` Obsidian :: Quickstart ::` prefix in your quickstart's pom.xml and regenerate it again. There is a JIRA asking to remove tha lt suffix from our quickstarts

**Comment by heiko-braun** [comment]
I've renamed the source repo according to @cmoulliard  and removed the `blabla` as requested by @gastaldi

**Comment by heiko-braun** [comment]
@cmoulliard I cannot run the archetype builder to complete this work. It always gets stuck at:

```
[...]
Cloning repo git://github.com/obsidian-toaster-quickstarts/rest_vertx_kube-configmap.git to /Users/hbraun/dev/prj/obsidian-quickstarts/platform/archetype-builder/../archetypes/../git-clones/rest_vertx_kube-configmap-archetype
```

**Pull request merged**